src/packages/@ncigdc/dux/bannerNotification.js

```diff
@@ -33,7 +33,7 @@ let initialState = [
   {
     components: ['PORTAL'],
     level: 'INFO',
-    id: 0,
+    id: 'initial_banner',
     dismissible: true,
     message: <DismissibleBanner />,
```

Comment:
might need `dismissed: false` so the `<DismissibleBanner />` still shows? not sure